pilot/pkg/networking/core/v1alpha3/accesslog.go

```diff
@@ -150,8 +150,12 @@ func (b *AccessLogBuilder) buildFileAccessLog(mesh *meshconfig.MeshConfig) *acce
 		if mesh.AccessLogFormat != "" {
 			formatString = mesh.AccessLogFormat
 		}
-		fl.AccessLogFormat = &fileaccesslog.FileAccessLog_Format{
-			Format: formatString,
+		fl.AccessLogFormat = &fileaccesslog.FileAccessLog_LogFormat{
+			LogFormat: &core.SubstitutionFormatString{
```

Comment:
This seems to be deprecated in v3 and v2 does not have this new config. So this needs a version check and the new config should be delivered only for 1.8 proxies.

Reply:
Do we need this? a v2 proxy cannot even connect anymore since we removed the translator

Reply:
1.7 proxy does not have these right?

Reply:
I am not sure, but if it does its not because `v2` vs `v3` right? the 1.7 proxy uses xds v3? But they could have added this field after 1.7

Reply:
Yeah. That is why I thought it is better to add that version check

Reply:
It is added in 1.15 == 1.7. 1.6 is already not compatible so I think its ok?

Reply:
Yeah. If 1.7 has it, version guard is not needed.